packages/grafana-ui/src/utils/statsCalculator.ts

```diff
@@ -31,10 +33,25 @@ export interface ColumnStats {
 // Internal function
 type StatCalculator = (data: SeriesData, fieldIndex: number, ignoreNulls: boolean, nullAsZero: boolean) => ColumnStats;
 
+/** Used to limit possible options */
+export type StatsFilter = (stat: StatCalculatorInfo) => boolean;
+
+/** Common Filters */
+export const StatsFilters = {
+  all: (stat: StatCalculatorInfo) => true,
+  numeric: (stat: StatCalculatorInfo) => stat.resultType === FieldType.number,
+
+  /** Some stats just pass the input value back.  Like `last` or `first` */
+  numericOrInput: (stat: StatCalculatorInfo) => {
+    return !stat.resultType || stat.resultType === FieldType.number;
+  },
```

Comment:
@torkelo -- ideas for a better name?  

The use case is for stats that really want a number (like gauges).  

Alternatively, this could filter for `not string` or something like that